Add enabled flag to Item (#1175)
PixarAnimationStudios/OpenTimelineIO: docs/tutorials/otio-serialized-schema.md

parameters:
- *effects*: 

> - *enabled*: Enabled sets whether or not an Item contributes to compositions. Analogous to Mute in various NLEs.

Reviewer: ```suggestion
- *enabled*: if true, an Item contributes to compositions. Analogous to Mute in various NLEs.
```

parameters:
- *effects*: 

> - *enabled*: Enabled sets whether or not an Item contributes to compositions. Analogous to Mute in various NLEs.

Reviewer: ```suggestion
- *enabled*: if true, an Item contributes to compositions. Analogous to Mute in various NLEs.
```

parameters:
- *effects*: 

> - *enabled*: Enabled sets whether or not an Item contributes to compositions. Analogous to Mute in various NLEs.

Reviewer: ```suggestion
- *enabled*: if true, an Item contributes to compositions. Analogous to Mute in various NLEs.
```

parameters:
- *effects*: 

> - *enabled*: Enabled sets whether or not an Item contributes to compositions. Analogous to Mute in various NLEs.

Reviewer: ```suggestion
- *enabled*: if true, an Item contributes to compositions. Analogous to Mute in various NLEs.
```

parameters:
- *effects*: 

> - *enabled*: Enabled sets whether or not an Item contributes to compositions. Analogous to Mute in various NLEs.

Reviewer: ```suggestion
- *enabled*: if true, an Item contributes to compositions. Analogous to Mute in various NLEs.
```

parameters:
- *effects*: 

> - *enabled*: Enabled sets whether or not an Item contributes to compositions. Analogous to Mute in various NLEs.

Reviewer: ```suggestion
- *enabled*: if true, an Item contributes to compositions. Analogous to Mute in various NLEs.
```
